diff --git a/node_modules/expo-dev-menu/vendored/react-native-reanimated/ios/LayoutReanimation/DevMenuREAUIManager.mm b/node_modules/expo-dev-menu/vendored/react-native-reanimated/ios/LayoutReanimation/DevMenuREAUIManager.mm
index ab7eb42..71e09d7 100644
--- a/node_modules/expo-dev-menu/vendored/react-native-reanimated/ios/LayoutReanimation/DevMenuREAUIManager.mm
+++ b/node_modules/expo-dev-menu/vendored/react-native-reanimated/ios/LayoutReanimation/DevMenuREAUIManager.mm
@@ -160,7 +160,17 @@ std::weak_ptr<devmenureanimated::Scheduler> _devMenuScheduler;
 - (RCTViewManagerUIBlock)uiBlockWithLayoutUpdateForRootView:(RCTRootShadowView *)rootShadowView
 {
   NSHashTable<RCTShadowView *> *affectedShadowViews = [NSHashTable weakObjectsHashTable];
-  [rootShadowView layoutWithAffectedShadowViews:affectedShadowViews];
+
+  // Create an NSPointerArray to hold the converted values
+  NSPointerArray *affectedShadowViewsArray = [NSPointerArray weakObjectsPointerArray];
+
+  // Iterate over the NSHashTable and add each object to the NSPointerArray
+  for (RCTShadowView *shadowView in affectedShadowViews) {
+      [affectedShadowViewsArray addPointer:(__bridge void * _Nullable)(shadowView)];
+  }
+
+  // Now pass the NSPointerArray to the method
+  [rootShadowView layoutWithAffectedShadowViews:affectedShadowViewsArray];
 
   if (!affectedShadowViews.count) {
     // no frame change results in no UI update block
